jinja/carnival/prize_booth.html

```diff
@@ -0,0 +1,76 @@
+{% extends 'base.html' %}
+{% block content %}
+{% block navbar %}
+{% include "navbar.html" %}
+{% endblock %}
+
+<div class="parallax" style="background-image: url('../img/xanadu-background.png'); box-shadow: none;">
+    <div class="container">
+
+        <div class="mt-5 pt-5 text-center teal-text text-lighten-2">
+            <img src="../img/carnival_logo_splash.png" class="logo"></img>
+        </div>
+        <div class="row-sm text-center justify-content-center" style="color:grey; font-family: 'Roboto', sans-serif !important;">
+            <h3 class="col-12 mt-5 mb-5 grey-text">
+                Exchange your tickets for amazing <strong>Carnival Prizes</strong>. 
+            </h3>
+            
+	<div style="font-size: 14pt;">
+            <p>
+            The participants with the most points will be the winners of some cool Xanadu swag. 😎

```

Comment:
Nathan's correct in that the words amazing/cool are overused here, even with the changes. I'd maybe change. "some cool Xanadu swag" to something else. "the newest Xanadu swag?" Something that says that some of these items are never seen before in the shop?

Reply:
I think cool the right word for this phrase. I removed it in the last phrase though so that we don't overuse it.

Reply:
```suggestion
            The participants with the most points will earn some cool Xanadu swag. 😎
```